examples/06-dc/plot_dipoledipole_2_5Dinversion_irls.py

```diff
@@ -0,0 +1,227 @@
+"""
+2.5D DC inversion of with Iterative Reweighted Least Squares
```

Comment:
Looking good @sgkang 
![image](https://user-images.githubusercontent.com/2406426/38281467-95e5a8b0-375f-11e8-95f0-50c113ec0144.png)

Reply:
One suggestion, for future examples, it would be nice to move moreso towards the notebook styled examples (eg. https://github.com/simpeg/simpeg/blob/master/examples/07-fdem/plot_loop_loop_2Dinversion.py)